.env.example

```diff
@@ -1,11 +1,10 @@
 NODE_ENV=production
-RIPPLED_WS_PORT=51233
-RIPPLED_RPC_PORT=51234
-RIPPLED_PEER_PORT=51235
-RIPPLED_HOST=your-rippled-host-url-here
+REACT_APP_RIPPLED_WS_PORT=51233
+REACT_APP_RIPPLED_PEER_PORT=51235
+REACT_APP_RIPPLED_HOST=your-rippled-host-url-here
```

Comment:
Why were these renamed to "REACT_APP_..."?

Reply:
You can only acess environment variables if they begin with `REACT_APP_....`